index.js

```diff
@@ -387,7 +387,7 @@ function getClusterProperties(cluster) {
 
 // longitude/latitude to spherical mercator in [0..1] range
 function lngX(lng) {
-    return lng / 360 + 0.5;
+    return lng / 360 + 0.5 + (lng > 180 || lng < -180 ? -Math.sign(lng) : 0);
```

Comment:
A slightly simpler / more reliable way to wrap the value:

```suggestion
    return ((lng / 360 + 0.5) % 1 + 1) % 1;
```

Reply:
Thanks for the tip on the alternative math. I tested it and unfortunately, it breaks the tests.

With the suggested approach, the returned values from `lngX` are different from the values from the first solution at their ~14-15 decimal digit, which causes the tests to fail. So I kept the original solution for now.

I've also added a test for this change in https://github.com/mapbox/supercluster/pull/194/commits/99c877f39d979bafbb0d2d3ae94ae0fc8364e39d.